db/system_keyspace.cc

```diff
@@ -66,23 +62,24 @@ namespace {
     const auto set_wait_for_sync_to_commitlog = schema_builder::register_static_configurator([](const sstring& ks_name, const sstring& cf_name, schema_static_props& props) {
         static const std::unordered_set<sstring> tables = {
             system_keyspace::PAXOS,
-            system_keyspace::BROADCAST_KV_STORE,
-            system_keyspace::TOPOLOGY,
-            system_keyspace::TOPOLOGY_REQUESTS,
-            system_keyspace::CDC_GENERATIONS_V3,
         };
         if (ks_name == system_keyspace::NAME && tables.contains(cf_name)) {
             props.wait_for_sync_to_commitlog = true;
         }
     });
     const auto set_use_schema_commitlog = schema_builder::register_static_configurator([](const sstring& ks_name, const sstring& cf_name, schema_static_props& props) {
```

Comment:
We'll want a single source of truth for this - one list of group 0 tables, and uniformly apply all group0 properties for them. For now this is ok.